rfcs/055-prismic-api/articles.md

```diff
@@ -0,0 +1,114 @@
+# Prismic API: articles endpoint
+
+## Example response
+
+```
+GET /articles/{id}
+{
+   {
+    "type": "Article",
+    "id": "XAEEIhQAACsA4eEl",
+    "format": {
+        "type": "Format",
+        "id": "W7TfJRAAAJ1D0eLK",
+        "name": "article"
+    },
+    "title": "Sick of being lonely",
+    "publicationDate": "2018-12-20T10:44:15+0000",
+    "caption": "When his relationship ended, Thom James first withdrew from the world, then began to suffer from illnesses with no apparent physical cause.",
+    "contributors": [
```

Comment:
Has this been cribbed from the Catalogue API or is it modelled like this in Prismic?

Reply:
it's a minimal version of both really, but closer to how things look in the catalogue API. In the raw prismic response we get:

```json
{
    "contributors": [
        {
            "role": {
                "id": "Wux6DyIAAO5n3lzk",
                "type": "editorial-contributor-roles",
                "tags": [],
                "lang": "en-gb",
                "slug": "artist",
                "first_publication_date": "2018-05-04T15:19:48+0000",
                "last_publication_date": "2019-08-14T14:31:04+0000",
                "link_type": "Document",
                "isBroken": false
            },
            "contributor": {
                "id": "Y8aT1REAAKREABkW",
                "type": "people",
                "tags": [],
                "lang": "en-gb",
                "slug": "alex-brenchley",
                "first_publication_date": "2023-01-17T12:27:22+0000",
                "last_publication_date": "2023-01-25T11:54:21+0000",
                "link_type": "Document",
                "isBroken": false
            },
            "description": []
        }
    ]
}
```

By making a couple of requests in the pipeline for those individual `people` and `editorial-contributor-roles`, i think we can tidy that up into something like

```json
{
    "contributors": [
        {
            "agent": {
                "id": "Y8aT1REAAKREABkW",
                "label": "Alex Brenchley",
                "type": "Person"
            },
            "roles": [
                {
                    "id": "Wux6DyIAAO5n3lzk",
                    "label": "Artist",
                    "type": "EditorialContributorRole"
                }
            ],
            "type": "Contributor"
        }
    ]
}
```

What do you think?

Reply:
- single role instead of an array
- `contributor` instead of `agent`